README.md

```diff
@@ -21,30 +20,30 @@ Demos
 
 ### Output sample from the RISC-V demo
 ```
-Testing DDR...                                                                                              
-Starting to fill with data                                                                                  
-Starting to read back data                                                                                  
-Passed                                                                                                      
-Loading payload                                                                                             
-Running payload                                                                                             
-ACEGHIJLMN                                                                                                  
+Testing DDR...
```

Comment:
how did this change?

Reply:
Oops :) I have a vim autocmd that removes extraneous spaces. Reverted that part.